web/themes/joinup/joinup_theme.theme

```diff
@@ -116,23 +142,70 @@ function joinup_theme_preprocess_fieldset(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_HOOK() for container.html.twig.
+ */
+function joinup_theme_preprocess_container(&$variables) {
+  $route_match = \Drupal::routeMatch();
+  $route_name = $route_match->getRouteName();
+  $form_routes = array(
+    'node.add',
+    'rdf_entity.rdf_add',
+    'entity.node.edit_form',
+    'collection.propose_form',
+    'entity.rdf_entity.edit_form',
+  );
+
+  // Check if container is field element - it has only one parent.
```

Comment:
There is more than one parent for example in wysiwyg editor elements. "Text format" dropdown and "About text formats" link have also form_wrapper class, so I needed different class for form field.